docs/UserManuals/ConfigUI/AdvancedMode.md

```diff
@@ -47,92 +81,114 @@ Below is an example for collecting 2 GitLab repos sequentially. It has 2 stages,
+Collect multiple Jenkins jobs sequentially. Below is an example for collecting 2 Jenkins jobs sequentially. It has 2 stages, each contains a Jenkins task.
 
 ```
 [
-  [
-    {
-      "Plugin": "github",
-      "Options": {
-        "repo": "lake",
-        "owner": "merico-dev"
-      }
-    },
-    {
-      "Plugin": "jira",
-      "Options": {
-        "connectionId": 1,
-        "boardId": 76
-      }
-    }
-  ]
+    [
+        {
+            "plugin": "jenkins",
+            "options": {
+                "connectionId": 1,
+                "scopeId": "auto_deploy"
+            }
+        }
+    ],
+    [
+        {
+            "plugin": "jenkins",
+            "options": {
+                "connectionId": 2,
+                "scopeId": "Deploy test"
+            }
+        }
+    ]
 ]
 ```
 
-### 3. Zentao
-Below is an example for collecting a Zentao workspace. Since users can configure multiple Zentao connection, it's required to pass in a `connectionId` for Tapd task to specify which connection to use.
+- `connectionId`: The `ID` field from **Connections Jenkins** page.
+- `scopeId`: Jenkins job name.
+
+### 4. Jira
+
+Collect multiple Jira boards sequentially. Below is an example for collecting 2 Jira boards sequentially. It has 2 stages, each contains a Jira task.
 
 ```
 [
-  [
-    {
-      plugin: 'zentao',
-      options: {
-        connectionId: 1,
-        productId: 1,
-        projectId: 1,
-        executionId: 1
-      }
-    }
-  ]
+    [
+        {
+            "plugin": "jira",
+            "options": {
+                "boardId": 8,
+                "connectionId": 1
+            }
+        }
+    ],
+    [
+        {
+            "plugin": "jira",
+            "options": {
+                "boardId": 26,
+                "connectionId": 1
+            }
+        }
+    ]
 ]
 ```
 
-- `connectionId`: The `ID` field from **ZENTAO Integration** page.
-- `productId`: optional, ZENTAO product id, see "Find Product Id" for details.
-- `projectId`: optional, ZENTAO product id, see "Find Project Id" for details.
-- `executionId`: optional, ZENTAO product id, see "Find Execution Id" for details.
+- `connectionId`: The `ID` field from **Connections Jira** page.
+- `boardId`: Just take a look at the URL - it will be the last number in the address. Should look something like this at the end: `RapidBoard.jspa?rapidView=8` or `/projects/xxx/boards/8`. So `8` would be the board ID in that case.
 
-You must choose at least one of `productId`, `projectId` and `executionId`.
+### 5. Jira + GitLab
 
-#### Find Product Id
-1. Navigate to the Zentao Product in the browser
-   ![](zentao-product.png)
-2. Click the red square annotated in the pic above
-   ![](zentao-product-id.png)
-3. Then the number in the red circle above is `ProductId`
+Below is an example for collecting a GitLab repo and a Jira board in parallel. It has a single stage with a GitLab task and a Jira task.
```

Comment:
It has a single stage with a GitLab task and a Jira task. As GitLab and Jira are using their own tokens, they can be executed in parallel.